### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by byronn-1**
Merge start of integration tests after first client review.

### CODE REVIEW

File: cypress/e2e/nhs-app.cy.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+
+const DISCLAIMER_TEXT =
+  "This application is for demo use only. It is not intended for real life use.";
+const STORAGE_AREA_TITLE = "Trauma Tower";
+const BOX_FOUR_TITLE = "Trauma Chest Drain - Box 4";
+
+describe("disclaimer pop-up", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
```
Comment (donal-s) [inline cypress/e2e/nhs-app.cy.js]: Can we change this to port 3000 and maybe extract to a const as it appears multiple times in this file?

**Review by donal-s — CHANGES_REQUESTED** [review]
We paired on this - one comment added

File: cypress/e2e/nhs-app.cy.js (hunk 1 above)
Comment (byronn-1) [reply cypress/e2e/nhs-app.cy.js]: Definitely, good spot!

Comment (byronn-1) [reply cypress/e2e/nhs-app.cy.js]: I changed the port number and attempted to add variable as suggested. Cypress threw an Error saying that you cannot call visit outside of a running test.

File: cypress/e2e/nhs-app.cy.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  it("clicking anywhere on gray screen does nothing", () => {
+    cy.root().click(100, 100);
+    cy.contains(DISCLAIMER_TEXT);
+  });
+});
+
+describe("storage area", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3002");
```
Comment (donal-s) [inline cypress/e2e/nhs-app.cy.js]: This still needs to be port 3000. I meant a const like:
`const SITE_URL = "http://localhost:3000";`

**Review by donal-s — CHANGES_REQUESTED** [review]

File: cypress/e2e/nhs-app.cy.js (hunk 1 above)
Comment (donal-s) [reply cypress/e2e/nhs-app.cy.js]: I meant just extract the string as a const, not the full line. Example in other comment.

Comment (byronn-1) [reply cypress/e2e/nhs-app.cy.js]: Ok will do!

File: cypress/e2e/nhs-app.cy.js (hunk 2 above)
Comment (byronn-1) [reply cypress/e2e/nhs-app.cy.js]: fixed! Please review

**Review by byronn-1 — COMMENTED** [review]
This is where you post new review comment?

File: cypress/e2e/nhs-app.cy.js (hunk 1 above)
Comment (donal-s) [reply cypress/e2e/nhs-app.cy.js]: Yup - fine now

File: cypress/e2e/nhs-app.cy.js (hunk 2 above)
Comment (donal-s) [reply cypress/e2e/nhs-app.cy.js]: All good

**Review by donal-s — APPROVED** [review]
Looks good

**Pull request merged**